HW/spark.py

```diff
@@ -0,0 +1,104 @@
+import sys
+
+from pyspark.sql import Row,SparkSession
+from pyspark.sql.types import (
+    BooleanType,
+    DoubleType,
+    IntegerType,
+    StringType,
+    StructField,
+    StructType,
```

Comment:
I don't think you need these imports because you never actually use them anywhere. Maybe we would need them if we saved a database locally after reading in a csv file or something.